lib/cloudfront_signer/distribution_registry.ex

```diff
@@ -5,16 +5,16 @@ defmodule CloudfrontSigner.DistributionRegistry do
   use Agent
   alias CloudfrontSigner.Distribution
 
-  def start_link() do
+  def start_link(_) do
```

Comment:
Why is it necessary to add a `_` here?

Reply:
I should clarify.
AFAIK (_)  means you are ignoring any passed params right?
Is this a style thing, or was there a specific reason for the change in this PR?
Otherwise it seems arbitrary (am I missing something?)

Reply:
I am using this PR as a bit of education. The previous code was deprecated as you can see by the changes I made in the Application module. And as I want to ignore any value passed in this function but the function now insists on being start_link/1 I use the underscore to indicate that to the compiler.

Reply:
`start_link/1` does the `/1` describe the number of arguments (a method signature of sorts?)

Reply:
Yep that is the convention in Erlang+Elixir